.vscode/settings.json

```diff
@@ -1,9 +1,5 @@
 {
-  "search.exclude": {
-    "**/.yarn": true,
-    "**/.pnp.*": true
-  },
-  "prettier.prettierPath": ".yarn/sdks/prettier/index.js",
-  "typescript.tsdk": ".yarn/sdks/typescript/lib",
+  "prettier.prettierPath": "node_modules/prettier/index.js",
+  "typescript.tsdk": "node_modules/typescript/lib",
```

Comment:
Was that intended?

Reply:
Yes! We're no longer using yarn so it broke the prettier and typescript